lms/services/moodle.py

```diff
@@ -83,7 +86,18 @@ def list_files(self, course_id: int):
                         )
                     )
 
-        return self._construct_file_tree(course_id, files)
+        file_tree = self._construct_file_tree(course_id, files)
+        self._file_service.upsert(
```

Comment:
We need to store files when we list them so we can use that info to fix course copy.